tenant/store.go

```diff
@@ -38,10 +40,11 @@ func (tenantStore *tenantStore) Entities() []interface{} {
 }
 
 type Tenant struct {
-	Id       uint64 `sql:"AUTO_INCREMENT"`
-	Name     string
-	Segments []Segment
-	Seq      uint64
+	ID         uint64 `sql:"AUTO_INCREMENT"`
+	ExternalID string `sql:"not null;unique"`
```

Comment:
Well, you said that you are treating UUIDs and IDs in the same way, allowing them to be used in the URL path. So, this query capability doesn't seem to be strictly necessary then.